### PR TITLE
Feature/localization

### PR DESCRIPTION
**Pull request by MRJCrunch**

### CODE REVIEW

File: generators/app/templates/app/libs/utils/localization.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict';
```
Comment (fxlrnrpt) [inline generators/app/templates/app/libs/utils/localization.js]: Why do you need a dedicated abstraction for localization? Just do configure step in middlewares initializer

File: generators/app/templates/app/libs/views.ejsSupport-addon/about.ejsSupport-addon.ejs [hunk 2]
```diff
@@ -6,9 +6,9 @@
         <h1><%%= message %%> ^ↀᴥↀ^</h1>
         <ul>
             <%% cats.forEach(function(cat) { %%>
-            <li><b>Name</b>: <%%= cat.name %%> </br>
-                <b>Boss</b>: <%%= cat.bossName %%> </br>
-                <b>Birth date</b>: <%%= cat.birthDate %%>
+            <li><b><%%=__('Name') %%></b>: <%%= cat.name %%> </br>
```
Comment (keenondrums) [inline generators/app/templates/app/libs/views.ejsSupport-addon/about.ejsSupport-addon.ejs]: You should pass localized strings here

File: generators/app/templates/app/libs/routes/api/v1/user.jwtSupport-addon.js [hunk 3]
```diff
@@ -160,19 +160,19 @@ async function verifyEmail(req, res) {
     let user = await models.User.findById(req.body.id);
 
     if (!user) {
-        throw new errors.NotFoundError('User not found.');
+        throw new errors.NotFoundError(req.__('%s is not found.', req.__('User')));
```
Comment (fxlrnrpt) [inline generators/app/templates/app/libs/routes/api/v1/user.jwtSupport-addon.js]: We should start using https://www.npmjs.com/package/messageformat to handle such messages

File: generators/app/templates/app/libs/routes/api/v1/user.jwtSupport-addon.js [hunk 4]
```diff
@@ -137,7 +137,7 @@ async function update(req, res) {
             });
 
             if (!user) {
-                throw new errors.NotFoundError('No user for request.');
+                throw new errors.NotFoundError(req.__('No user for request.'));
```
Comment (fxlrnrpt) [inline generators/app/templates/app/libs/routes/api/v1/user.jwtSupport-addon.js]: Why can't you use {{target}} not found message here?

File: generators/app/templates/app/libs/locales/en.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+	"Name": "Name",
+	"%s must be from 1 to 255 symbols.": "%s must be from 1 to 255 symbols.",
+	"Boss name": "Boss name",
+	"Cat": "Cat",
+	"%s is not found.": "%s is not found.",
```
Comment (fxlrnrpt) [inline generators/app/templates/app/libs/locales/en.json]: Don't use positional substitutes, always use named substitues, e.g. {{target}} not found

**Review by fxlrnrpt — CHANGES_REQUESTED** [review]

File: generators/app/templates/app/libs/routes/api/v1/security.jwtSupport-addon.js [hunk 6]
```diff
@@ -33,15 +33,15 @@ async function login(req, res) {
     });
 
     if (!user) {
-        throw new errors.SecurityError('Login and password combination is not found.');
+        throw new errors.SecurityError(req.__mf('Login and password combination is not found.'));
```
Comment (fxlrnrpt) [inline generators/app/templates/app/libs/routes/api/v1/security.jwtSupport-addon.js]: You can use the same {value} is not found pattern here

**Review by fxlrnrpt — CHANGES_REQUESTED** [review]

**Review by fxlrnrpt — APPROVED** [review]

**Pull request merged**